kuttl-tests/kind-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly script_directory="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+readonly project_directory="$(readlink -f "${script_directory}/..")"
+
+# shellcheck source=../metadata.sh
+source "${project_directory}/metadata.sh"
+
+KUBECTL_VERSION=1.18.4
+KUTTL_VERSION=0.5.0
+KUBECTL_KUDO_VERSION=${DS_KUDO_VERSION#v}
+KIND_VERSION=0.8.1
+
+ARTIFACTS=kuttl-dist
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+KUDO_MACHINE=$(uname -m)
+MACHINE=$(uname -m)
+if [ "${MACHINE}" == "x86_64" ]; then
+  MACHINE=amd64
+fi
+
+mkdir -p bin/
+
+curl -Lo "bin/kubectl_${KUBECTL_VERSION}_${OS}" "https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/${OS}/${MACHINE}/kubectl"
```

Comment:
this will download on each run. Why not keep them in Makefile targets? and maybe fix them if they were being downloaded on each run

Reply:
Makefiles aren't the right tool for the job here. This is more about batching, less about complex dependency management. Furthermore, Makefiles aren't great at handling environment variables and other script related things -- even though it's not impossible. And we already have a defined way of setting up the environment for scripts with `metadata.sh` and should keep using this here.
Good point about the downloads though, let me add some checks for existing binaries.